models/orderDao.js

```diff
@@ -0,0 +1,157 @@
+const { dataSource } = require('./dataSource');
+
+const getOrderStatusId = async (orderStatus) => {
+  try {
+    const [orderStatusId] = await dataSource.query(
+      `
+          SELECT id
+              FROM order_status
+              WHERE status = ?
+          `,
+      [orderStatus]
+    );
+    return orderStatusId;
+  } catch (error) {
+    error = new Error('DATABASE_CONNECTION_ERROR');
+    error.statusCode = 400;
+    throw error;
+  }
+};
+
+const completeOrders = async (
+  userId,
+  orderNumber,
+  address,
+  netPoint,
+  SubscribeDeliveryTime,
+  carts
+) => {
+  const queryRunner = dataSource.createQueryRunner();
+
+  await queryRunner.connect();
+  await queryRunner.startTransaction();
+
+  const orderStatusId = await getOrderStatusId('배송준비중');
```

Comment:
- 상태값이 한글인가요? 데이터를 검색하는 왠만하면 영어로 하는 것이 좋습니다.
- 위의 리뷰에서 enum화 한 것을 다음과 같이 사용하면 됩니다.
```suggestion
  const orderStatusId = orderStatusId['pendding'];
```